docs/UserGuide.md

```diff
@@ -3,271 +3,478 @@ layout: page
 
-Format: `lock [CURRENT_PASSWORD] [NEW_PASSWORD]`
+**Purpose**: Locks ClientBook with a user-selected password.
+
+**Format**: `lock [CURRENT_PASSWORD] [NEW_PASSWORD]`
 
 * Verifies the current password before locking ClientBook with the new password.
 * The `CURRENT_PASSWORD` field can be omitted if ClientBook is not yet locked.
 * When `CURRENT_PASSWORD` and `NEW_PASSWORD` fields are both omitted, ClientBook will attempt to lock itself using the last used password that is safely stored on your device.
 
-### Unlocking ClientBook : `unlock`
+:exclamation: **Note**: After setting a password, application can only be launched through Command Prompt or Terminal.
+  * Refer to [Quick Start](#quick-start) on how to launch the application through Command Prompt or Terminal.
+
+**Examples**:
+* Lock ClientBook with password `123`.
+  * `lock 123`
+
+    ![lock 123](images/lock-new-pw.png)
+
+* Change ClientBook password from `123` to `456`.
+  * `lock 123 456`
+
+    ![lock 456](images/lock-update-pw.png)    
+
+[Return to Table of Contents](#table-of-contents)
 
-Unlocks ClientBook.
 
-Format: `unlock CURRENT_PASSWORD`
+### Unlock ClientBook : `unlock`
+
+**Purpose**: Unlocks ClientBook.
+
+**Format**: `unlock CURRENT_PASSWORD`
 
 * Verifies the current password before unlocking ClientBook.
+* Password is removed after executing this command. There is no need for to enter the password to launch the application.
 
+**Examples**:
+* Unlock ClientBook with wrong password `123`.
+    * `unlock 123`
 
-### Saving the data
+      ![unlock 123](images/unlock-wrong-pw.png)
 
-ClientBook saves its data in the hard disk automatically after any command that changes the data. There is no need to save manually.
+* Unlock ClientBook with correct password `456`.
+    * `unlock 456`
 
-### Editing the data file
+      ![unlock 456](images/unlock-correct-pw.png)
 
-ClientBook saves its data as a JSON file `[JAR file location]/data/clientbook.json`. Advanced users are welcome to update data directly by editing that data file.
+[Return to Table of Contents](#table-of-contents)
 
-<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-If your changes to the data file makes its format invalid, ClientBook will discard all data and start with an empty data file at the next run.
-</div>
 
-### Archiving data files `[coming in v2.0]`
+### Exiting the program : `exit`
+
+**Purpose**: Exits the program.
+
+**Format**: `exit`
+
+[Return to Table of Contents](#table-of-contents)
+
+
+### Saving data
+
+**Purpose**: ClientBook saves its data to a file in the home folder automatically after any command that changes the data. There is no need to save manually!
+
+[Return to Table of Contents](#table-of-contents)
+
+
+### Editing data file
+
+**Purpose**: ClientBook saves its data as a JSON file `[JAR file location]/data/clientbook.json`. Advanced users are welcome to update data directly by editing that data file.
```

Comment:
Currently the json is stored in a zip file regardless of whether it is locked. If its locked then there's a password to the zip file.

Reply:
Okay, I've added the below 2 statements for clarity:

* The data file is stored in a zip file inside the `data` folder in the same folder. 
* If you previously set a `lock` for ClientBook, the zip folder can be unzipped with that same password.

Reply:
By the way, I tried to update the data directly from the json file, but not sure if it is because of the zip, but it doesn't work. After I change the json file, I start up the app and it goes back to the previous state. Maybe you guys can test it also, and if it doesn't work, we just remove the "Advanced users are welcome to update data directly by editing that data file." part